### PR TITLE
Added instructions for setting up an H2 datasource

### PR DESCRIPTION
**Pull request by sgilda**
Also made the Byteman halt more distinct.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -317,6 +319,88 @@ The following procedures describe how to add a user with the appropriate permiss
...
+                        <background-validation> false </background-validation>
+                        <background-validation-millis> 0 </background-validation-millis>
+                    </validation>
+                    <statement>
+                        <prepared-statement-cache-size> 0 </prepared-statement-cache-size>
+                        <share-prepared-statements> false </share-prepared-statements>
+                    </statement>
+                </xa-datasource>
+
+
```
Comment (pmuir) [inline README.md:403]: These instructions could be easily generalised to be for any datasource.

Though, I am wondering if we wouldn't be better off using a deployable datasource.

**Comment by pmuir** [comment]
Otherwise looks good.

File: README.md (hunk 1 above)
Comment (sgilda) [inline README.md:403]: Sorry. I'm unclear what I need to do here. :-(

**Comment by sgilda** [comment]
Should I remove the H2 datasource instructions?

**Comment by sgilda** [comment]
Closing this pull. Will replace it with something else later.

**Pull request closed without merging**